README.md

```diff
@@ -19,58 +19,45 @@ After cloning the smartwallet-app repository to a directory on your computer, en
 yarn install
 ```
 
-### Browser
+### Device Build
 
-#### With hot module reloading support
+#### Android
 
-```bash
-yarn start
-```
-This will make webpack-dev-server serve the application on https://localhost:8080 and will reload only the modules you've modified instead of rebuilding the entire bundle on every change. The changes will be applied live to your browser.
+Please set up an Android development environment. 
```

Comment:
I would rename it to React Native Android development env - instructions for it are slightly different.